### PR TITLE
Reduce duplicated code

### PR DESCRIPTION
**Pull request by AlexHladin**
## Motivation
Reduce duplicated code. Duplicated code could cause issues related to the missed bug fixes in some place of the source code.

## Changes
Moved duplicated code to separate functions.

## Related

(Optional, link(s) of related Issues or PR related to this PR, If this PR adds or changes functionality, please take some time to update the docs and link your PR here.)

## Additional Comments

(Anything, optional)

### CODE REVIEW

**Review by kimlimjustin — CHANGES_REQUESTED** [review]
Overall, it's a nice catch! 

For the `>` and `<` operator, do you want to work on it (I think passing an argument to the function will make it works), It's okay if you are unavailable, I can add it.

Thanks and Regards

File: src/Components/Files/File Operation/select.ts [hunk 1]
```diff
@@ -301,20 +267,9 @@ const Initializer = () => {
...
 			}
 		} else if (e.key === 'ArrowUp' && !e.altKey) {
-			if (!document.contains(latestSelected)) {
-				selectFirstFile();
-				return;
-			}
-			if (
-				Array.from(latestSelected.parentNode.children).indexOf(
-					latestShiftSelected
-				) >
```
[the remark this answers is not in the document]
Comment (AlexHladin) [reply src/Components/Files/File Operation/select.ts]: @kimlimjustin fixed

**Comment by kimlimjustin** [comment]
IDK what's went wrong but it breaks the file section, lemme fix it

File: src/Components/Files/File Operation/select.ts [hunk 2]
```diff
@@ -108,15 +94,9 @@ const Initializer = () => {
...
 			}
-			if (
-				Array.from(latestSelected.parentNode.children).indexOf(
-					latestShiftSelected
-				) <
-				Array.from(latestSelected.parentNode.children).indexOf(
-					latestSelected
-				)
-			)
+			if (childIndex(latestSelected.parentNode) < childIndex(latestSelected.parentNode)) {
```
Comment (kimlimjustin) [inline src/Components/Files/File Operation/select.ts:97]: I figured out why this broke the function, 

let a = `childIndex(latestSelected.parentNode)`, then
`if(a < a)` should always return false

**Review by kimlimjustin — CHANGES_REQUESTED** [review]
Don't mixed up `latestSelected` with `latestShiftSelected` :) 

I think I'll fix this.

**Review by kimlimjustin — APPROVED** [review]

File: src/Components/Files/File Operation/select.ts (hunk 2 above)
Comment (AlexHladin) [reply src/Components/Files/File Operation/select.ts]: Good catch! I did not see this issue :(

Comment (kimlimjustin) [reply src/Components/Files/File Operation/select.ts]: Haha, I wasted almost an hour to figure it out lmao, thank you so much tho.

If you got time, please help me refactor my code haha (some of them are so ugly, but not contextmenu cuz I'll surely refactor whole of it), or maybe help contribute by picking any issue there 

TYSM

Comment (AlexHladin) [reply src/Components/Files/File Operation/select.ts]: Will do my best!

**Pull request merged**